assets/scss/comments-and-meta.scss

```diff
@@ -7,11 +7,11 @@
   margin-bottom: 1rem;
   .comment-list {
     margin-bottom: 1rem;
-    background-color: #f0f4f5;
+    background-color: $color-nhsuk-light-grey;
     .nhsuk-list-panel__item {
       padding: 1.5rem 0;
       display: block;
-      background-color: #f0f4f5;
+      background-color: $color-nhsuk-light-grey;
       #respond {
```

Comment:
No chance of moving these styles to a class rather than an ID?

Reply:
WP inserts the id. And it is a unique form, so if you open a new response box it is actually the same one. Using ID in the css keeps it sane.